Obsidian/Commands/MainCommandModule.cs

```diff
@@ -363,6 +367,23 @@ public async Task StopAsync(CommandContext Context)
                 server.StopServer();
             });
         }
+        #endregion
+
+        #region time
+
+        [Command("time")]
+        [CommandInfo("Sets declared time", "/time <timeOfDay>")]
+        public async Task TimeAsync(CommandContext Context) => TimeAsync(Context, 1337);
+        [CommandOverload]
+        public async Task TimeAsync(CommandContext Context,int time)
+        {
+            var player = Context.Player as Player;
+            player.client.SendPacket(new TimeUpdate(0, time));
```

Comment:
I think this should be moved to IWorld to allow usage from plugins

Reply:
Good point! The command will be changed anyways, since commands are broken now (I will open an issue for it). It's more or less a test command right now.

Reply:
I'd like to point out that the time packet here is not being broadcasted, it's only being sent to the executing client.

Reply:
+ if we merge it it might be nice to:
A. track time internally
B. send time to client when it joins.